From bb0aabf7e85aa4dd46de3782737bb6327693db68 Mon Sep 17 00:00:00 2001
From: Ted Ross <tross@apache.org>
Date: Mon, 16 May 2011 21:17:57 +0000
Subject: [PATCH 5/5] Bug 692134 - Help/Usage text for qpid-config is
 incorrect

NO-JIRA - Fixed another problem in the help text.

git-svn-id: https://svn.apache.org/repos/asf/qpid/trunk@1103897 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 9063dbfb3fa9cd723416f4c6cad7872f084f8a44)
---
 qpid/tools/src/py/qpid-config |   10 +++++-----
 1 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/qpid/tools/src/py/qpid-config b/qpid/tools/src/py/qpid-config
index a255441..3df69dc 100755
--- a/qpid/tools/src/py/qpid-config
+++ b/qpid/tools/src/py/qpid-config
@@ -41,15 +41,15 @@ Usage:  qpid-config [OPTIONS]
 description = """
 ADDRESS syntax:
 
-      [username/password@] hostname
-      ip-address [:<port>]
+      [username/password@] hostname [:<port>]
+      [username/password@] ip-address [:<port>]
 
 Examples:
 
 $ qpid-config add queue q
-$ qpid-config add exchange direct d localhost:5672
-$ qpid-config exchanges 10.1.1.7:10000
-$ qpid-config queues guest/guest@broker-host:10000
+$ qpid-config add exchange direct d -a localhost:5672
+$ qpid-config exchanges -a 10.1.1.7:10000
+$ qpid-config queues -a guest/guest@broker-host:10000
 
 Add Exchange <type> values:
 
-- 
1.7.4.4

